src/org/opendatakit/briefcase/ui/reused/TableCustomizer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.ui.reused;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import javax.swing.JTable;
+import javax.swing.UIManager;
+import javax.swing.table.JTableHeader;
+
+public class TableCustomizer {
+
+  /**
+   * for header customization.
```

Comment:
I'd suggest to change this to something that explains what is this method doing. Something like "Sets properties of the given {@link JTableHeader} to improve its visual appearance across all platforms (Mac, Windows, and Linux)"

Reply:
Improved.